R/plots.R

```diff
@@ -100,12 +105,12 @@ plot.GeoLiftPower <- function(x,
         PowerPlot_aux <- as.data.frame(PowerPlot %>% dplyr::filter(duration == dur))
 
         CostPerLift <- as.numeric(x %>%
-          dplyr::filter(duration == dur, lift > 0) %>%
-          dplyr::mutate(AvgCost = investment / lift) %>%
-          dplyr::summarise(mean(AvgCost)))
+                                    dplyr::filter(duration == dur, lift > 0) %>%
+                                    dplyr::mutate(AvgCost = investment / lift) %>%
+                                    dplyr::summarise(mean(AvgCost)))
 
         PowerPlot_graph <- ggplot(PowerPlot_aux, aes(x = lift, y = power)) +
-          geom_smooth(formula = y ~ x, color = "indianred3", method = "loess", se = FALSE) +
+          geom_smooth(formula = y ~ x, color = "#52854C", method = "loess", se = FALSE) +
```

Comment:
Could we add the smoothing curve as an optional hyperparameter to the function?

Reply:
Changed the default of `actual_values` to TRUE